apps/ehr/.lintstagedrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "*.(tsx|ts|js|jsx)": ["cross-env NODE_OPTIONS='--max-old-space-size=8192' eslint --fix", "prettier --write"]
+  "*.(tsx|ts|js|jsx)": [
+    "cross-env NODE_OPTIONS='--max-old-space-size=8192' eslint --report-unused-disable-directives --max-warnings 0 --fix",
```

Comment:
https://eslint.org/docs/latest/use/command-line-interface#--report-unused-disable-directives-severity
`--report-unused-disable-directives` will only be warnings not errors, just fyi since `max-warnings 0` addresses that